websites/S/Stan/presence.ts

```diff
@@ -22,6 +23,10 @@ const presence = new Presence({
     meta?: {
       [key: string]: string;
     };
+    strings: Awaited<ReturnType<typeof getStrings>>;
+    coverUrls?: {
+      [key: string]: string;
+    };
     settings?: {
       id?: string;
       delete?: boolean;
```

Comment:
Again, up to you if you wanna fix this in this PR but you should probably not use reserved keywords as setting names or properties